classes/services/queryBuilders/PKPPublicationQueryBuilder.inc.php

```diff
@@ -120,8 +120,8 @@ public function get() {
 	public function getDateBoundaries() {
 		$q = $this->get();
 		$q->select([
-			'MIN(p.date_published)',
-			'MAX(p.date_published)'
+			Capsule::raw('MIN(p.date_published)',
+			'MAX(p.date_published)')
```

Comment:
Thanks, can you put it all on one line or separate out the `Capsule` statement from the selection?

```
Capsule::raw('MIN(p.date_published)', 'MAX(p.date_published)')
```

Or:

```
Capsule::raw(
	'MIN(p.date_published)',
	'MAX(p.date_published)'
)
```